stage1/init/init.go

```diff
@@ -150,6 +152,19 @@ func machinedRegister() bool {
 	return found == 2
 }
 
+func copySystemd(p *Pod) error {
+	systemdBin := []string{
+		"/lib/systemd/systemd:/lib/systemd/systemd",
+		"/usr/bin/systemctl:/bin/systemctl",
```

Comment:
oh, I didn't know about LookPath... I kind of reimplemented it but in a more generic way so it works for `/lib/systemd/*`. See the code I just pushed. What do you think of it?